### PR TITLE
historic app activity and apps separated

### PR DESCRIPTION
**Pull request by ChriLnth**
## Changelog

- App history doesnt delete when disconnecting from app
- Now shows Unplugged from when disconnecting 

### Type of changes included:

- [ ] Components
- [x] Business Logic
- [ ] Bug fixes
- [ ] Styling
- [x] Code Refactor

### Clubhouse Tickets Related:

- [improvement-suggestion-activity-tab](https://app.shortcut.com/terminalsystems/story/24510/improvement-suggestion-activity-tab)

### Screenshots/Gifs:
image

### Tested on:

- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by aleacevedo — APPROVED** [review]

File: source/Background/Controller.js [hunk 1]
```diff
@@ -149,9 +149,28 @@ backgroundController.exposeController('disconnect', async (opts, url) => secureC
...
+            ...apps[url].events,
+            {
+              status: CONNECTION_STATUS.disconnected,
+              date,
+            }
+          ]
+        },
+      };
       storage.set({ [keyring.currentWalletId]: { apps: newApps } });
+
```
Comment (tomiir) [inline source/Background/Controller.js:173]: I'd remove the comment and make all of this a func like `addDisconnectEntry` or sth like that just to keep it similar to how it was.

File: source/Background/Controller.js [hunk 2]
```diff
@@ -186,6 +208,14 @@ backgroundController.exposeController(
...
           icon: icons[0] || null,
           timeout,
+          date,
+          events: [
+            ...response?.[keyring.currentWalletId]?.apps[domainUrl].events,
+            {
+              status: CONNECTION_STATUS.pending,
+              date,
+            }
+          ],
```
Comment (tomiir) [inline source/Background/Controller.js:218]: Do we want to log this in the events?

File: source/hooks/useApps.jsx [hunk 3]
```diff
@@ -27,17 +43,39 @@ const useApps = () => {
...
+          status: event.status,
+          icon: app.icon,
+          name: app.name,
+          url: app.url,
+          whitelist: app.whitelist
+        }
+      })
+    );
+
+    console.log('allEvents', allEvents);
```
Comment (tomiir) [inline source/hooks/useApps.jsx:66]: remove

File: source/hooks/useApps.jsx [hunk 4]
```diff
@@ -27,17 +43,39 @@ const useApps = () => {
...
+    const historic = allEvents.filter((a) => a.status === CONNECTION_STATUS.accepted || CONNECTION_STATUS.disconnected);
+
     setParsedApps(filtered);
+    setHistoricApps(historic);
   }, [apps]);
 
   return {
     parsedApps,
     removeApp: handleRemoveApp,
+    historicApps,
```
Comment (tomiir) [inline source/hooks/useApps.jsx:78]: what's parsedApps for ?

**Review by tomiir — CHANGES_REQUESTED** [review]

File: source/Background/Controller.js (hunk 2 above)
Comment (ChriLnth) [reply source/Background/Controller.js:218]: idk, i put it there just to keep consistency, but we dont really use it

Comment (ChriLnth) [reply source/Background/Controller.js:218]: ill remove it

File: source/hooks/useApps.jsx (hunk 4 above)
Comment (ChriLnth) [reply source/hooks/useApps.jsx:78]: its parsed + filtered (only accepted) apps

**Review by rocky-fleek — APPROVED** [review]

**Pull request merged**